src/Equinor.Procosys.Library.Query/GetFunctionalRoles/GetFunctionalRolesQuery.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using MediatR;
+using ServiceResult;
+
+namespace Equinor.Procosys.Library.Query.GetFunctionalRoles
+{
+    public class GetFunctionalRolesQuery : IRequest<Result<IEnumerable<FunctionalRoleDto>>>
+    {
+        public GetFunctionalRolesQuery(string plant, string classification)
+        {
+            Plant = plant ?? throw new ArgumentNullException(nameof(plant));
```

Comment:
Is this a check for a validator? I guess if plant is null, we will never we able to get to a point where this class is initiated?

Reply:
True, will remove